types/migrations/20220518174701_edited_collections.sql

```diff
@@ -0,0 +1,28 @@
+-- Add migration script here
+
+-- A file for collections and chapters
+
+create extension if not exists "ltree";
+
+create table collections (
+  id autouuid primary key,
+  collection_name text not null,
+  collection_desc text,
```

Comment:
Hmm we don't need the prefixes here, I'd go for `title` and I'm not positive that we need description text yet because that might be Wordpress content.